src/collection-browser.ts

```diff
@@ -162,6 +162,12 @@ export class CollectionBrowser
    */
   private endOfDataReached = false;
 
+  /**
+   * When the page transitions from desktop to mobile, we need to
+   * disable the transition animation.
+   */
+  private mobileFiltersloading = false;
```

Comment:
suggestion: `mobileFiltersLoading`
also same question on using `is` prefix here